vulnerabilities/helpers.py

```diff
@@ -267,3 +267,16 @@ def _get_gh_response(gh_token, graphql_query):
     endpoint = "https://api.github.com/graphql"
     headers = {"Authorization": f"bearer {gh_token}"}
     return requests.post(endpoint, headers=headers, json=graphql_query).json()
+
+
+def dedupe(original: List) -> List:
```

Comment:
You could also have implemented this using a `OrderedDict` from `collections`.

```
>>> import collections
>>> collections.OrderedDict.fromkeys(["z","i","a","a","d","d"]).keys()
odict_keys(['z', 'i', 'a', 'd'])
```

Reply:
good point! actually on Python 3.6+, this could be even shorter: `dict.fromkeys(["z","i","a","a","d","d"]).keys()`